syntax/mediafile.php

```diff
@@ -138,7 +139,7 @@ public function render($format, Doku_Renderer $renderer, $data)
                 $imageAttributes['data-pngcache'] = str_replace([$conf['cachedir'], Diagrams::CACHE_EXT], '', $cachefile);
             }
 
-            $image = sprintf('<object %s></object>', buildAttributes($imageAttributes, true));
+            $image = sprintf('<object %s><span class="diagrams-alt">' . $wrapperAttributes['title'] . '</span></object>', buildAttributes($imageAttributes, true));
```

Comment:
The title needs to be escaped here. (we don't do it for the other attributes because buildAttributes() takes care of that)

Reply:
Fixed.